### PR TITLE
[NIFI-13349] align angular material and tailwind typography

### PR DESCRIPTION
**Pull request by scottyaslan**
Overall this change should not produce many differences. The default font-size is 14px with a 20px line-height and 400 weight. This PR updates the tailwind fontSize configuration to match those of the angular material typography such that the tailwind styles may be used alongside the angular typography.

### CODE REVIEW

**Review by rfellows — COMMENTED** [review]
@scottyaslan Just noticed a couple of things with the username/logout display. Not opposed to making the text larger than before, just not sure it was intended.

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/apps/nifi/src/app/ui/common/navigation/navigation.component.scss [hunk 1]
```diff
@@ -21,7 +21,6 @@
     }
 
     .current-user {
-        font-size: 12px;
```
[the remark this answers is not in the document]
Comment (scottyaslan) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/apps/nifi/src/app/ui/common/navigation/navigation.component.scss:24]: Yes, in old nifi 12px was basically the default font size. Table text, dialog text, menu text, etc. all used 12px. In the new UI the default font is 14px so I think this change is correct and intentional.

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/apps/nifi/src/app/ui/common/navigation/navigation.component.scss [hunk 2]
```diff
@@ -30,7 +29,6 @@
     }
 
     a {
-        font-size: 12px;
```
[the remark this answers is not in the document]
Comment (scottyaslan) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/apps/nifi/src/app/ui/common/navigation/navigation.component.scss:33]: Same comment as the`.current-user`. I think 14px is accurate for the new UI.

**Review by rfellows — APPROVED** [review]
Good work @scottyaslan. Thanks!

**Pull request merged**